pom.xml

```diff
@@ -376,7 +375,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.13</version>
+        <version>2.22.2</version>
```

Comment:
Yeah, here's a case where you've grabbed the latest `2.x` release rather than a `3.0.0-Mx` release. I'm inclined to stick with the latest `2.x` both here and in the previous case, although in this case it's an easier choice because the `2.22.2` release was just 1 month ago rather than 5 years!

https://search.maven.org/search?q=g:org.apache.maven.plugins%20AND%20a:maven-surefire-plugin&core=gav

Reply:
Let's stick to `2.22.2` until `3.x` is GA.